### ISSUE TITLE
nbconvert to notebook doesn't respect line endings

### ISSUE THREAD

**Issue by jamesmyatt** [jupyter/nbconvert#1062]
On Windows, when I run

    jupyter nbconvert --to notebook --execute --inplace "Untitled25.ipynb"

it uses CRLF line endings, but when when I open and save it directly via Jupyter notebook or Jupyterlab it always saves with LF line endings.

I think that the best solution would be to match the behaviour of Jupyter notebook, i.e. replace the line endings with LF. Alternatively, it should maintain the existing line endings, rather than (presumably) always using "native".

nbconvert 5.5.0, notebook 5.7.8, pandoc 2.7.3, jupyerlab 1.0.1

**Comment by mydmdm** [comment]
same issue, I write a script to clear cell outputs before committing for security, but the command 
```
jupyter nbconvert --ClearOutputPreprocessor.enabled=True --inplace
```
changes the line ending

**Comment by MSeal** [comment]
If we change this behavior we should leave a flag to preserve system line-endings. I'd be ok with someone making that change so long as it could be opt-ed out of. There's sometimes system specific actions that complain when line-endings don't match the system but most modern windows applications and tools handle it smoothly now-a-days.

**Comment by mydmdm** [comment]
Maybe a option flag to let user select line ending among 
- keep-as-it-is (the same to the source file)
- auto (depends on the OS running the command)
- LF
- CRLF

Agreed that the default value of such flag would preserve the current behavior (now is auto?)

**Comment by MSeal** [comment]
Yes. I believe it is auto atm. I'd also be ok changing the default to `LF` in the 6.x branch if we wanted as conforming to other notebook executions might be nice and the major change is easy to justify default changes.

**Comment by jamesmyatt** [comment]
The default should be the same as the Jupyter notebook or lab editors. It's a mistake that it's not the same and I can't imagine how anyone would be happy that the line endings change. I'm happy for it to be an option, but the default should be the one the creates the least issues. If no-one has raised this as an issue before, then it's probably because they're not looking.

**Comment by MSeal** [comment]
I added it to the 6.0 milestone -- could use some extra contributors helping get small issues like these moved across the finish line before the last major issues get resolved.

**Comment by auxym** [comment]
Hi, any update on this? I'm running nbconvert 6.0.0 and still experiencing the issue. Is the fix included in a more recent version?

**Comment by Grufoony** [comment]
Has this issue anything to do with enumeration? I think I found a bug in which you have to manually insert a new line at the end of an enumeration to actually have it to go on a new line in the pdf.

For example if you write this markdown
```
My enum:
- item1
- item2
- ...
```
in the pdf will appear on a single line, but if you write this
```
My enum:

- item1

- item2

- ...
```
it is correctly converted
